contracts/Pool.sol

```diff
@@ -37,18 +55,31 @@ contract Pool is IPool {
     }
 
     event AddedAdmin(address _address);
-
+    
     event RemovedAdmin(address _adddress);
+    
+    event AddedAuditReport(AuditorReports.Audit _audit);
 
-    constructor (string memory _name, address[] memory _admins) {
-        name = _name;
+    constructor (
+        string memory _name,
+        address _governer,
```

Comment:
this constructor is currently invoked directly by the UI. I dont think user should care about which governer is assigned to which pool.